Source/Core/VideoCommon/BPMemory.h

```diff
@@ -949,6 +1327,19 @@ union FieldMask
   BitField<1, 1, bool, u32> even;
   u32 hex;
 };
+template <>
+struct fmt::formatter<FieldMask>
+{
+  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  template <typename FormatContext>
+  auto format(const FieldMask& mask, FormatContext& ctx)
+  {
+    return format_to(ctx.out(),
+                     "Enable odd field: {}\n"
+                     "Enable even field: {}",
+                     mask.odd ? "Yes" : "No", mask.even ? "Yes" : "No");
```

Comment:
These descriptions might be backwards based on the comment above:

https://github.com/dolphin-emu/dolphin/blob/950cfcec9bd8c3253b11bca64426b3886c5456dc/Source/Core/VideoCommon/BPMemory.h#L1323-L1329

Reply:
They weren't backwards (I misread the comment), but I've created an enum for these fields (and for `FieldMode`) to avoid confusion in the future.